docs/data-management/mriqc.md

```diff
@@ -73,12 +74,18 @@ In addition, *MRIQC* is executed prior any further processing step considering o
         - [ ] Address the issue (e.g., out-of-memory when running a container) and re-run *MRIQC*.
             Proceed as in the previous case by finding documentation on the *MRIQC* repository or NeuroStars.
 
-- [ ] Visualize the first mosaic (background mosaic) and apply the [exclusion criteria](qaqc-criteria.md#view-of-the-background-of-the-anatomical-image).
-- [ ] Scroll down to the zoom in the zoomed-in brain mosaic and apply the [exclusion criteria](qaqc-criteria.md#zoomed-in-mosaic-view-of-the-brain).
+- [ ] Visualize the first mosaic (background mosaic) and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#view-of-the-background-of-the-anatomical-image).
+
+!!! warning "Running through the visual report does not have to be executed in a fixed order"
+
+    While we describe the run through the report as linear, often inspecting other reportlets can help make a decision about the exclusion criteria related to the reportlet at hand. As such, we encourage you to jump back and forth between visualizations as much as needed.
+
+- [ ] Scroll down to the zoom in the zoomed-in brain mosaic and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#zoomed-in-mosaic-view-of-the-brain).
+- [ ] Verify that no error is reported in the section "About > Errors". If there is, follow the procedure described in "*MRIQC* failed to produce all the expected visual reports".
```

Comment:
Also, it would be nice to add an anchor right before or inside the admonition block and convert that `"*MRIQC* failed to produce all the expected visual reports"` into a link.